framework/src/modules/pos/endpoint.ts

```diff
@@ -347,4 +354,17 @@ export class PoSEndpoint extends BaseEndpoint {
 			getWaitTime(callerAddress, validatorAddress) + unstakeHeight,
 		);
 	}
+
+	private _calculatePunishmentPeriods(pomHeights: number[], period = PUNISHMENT_PERIOD) {
```

Comment:
I think second params is never used, so it's ok to just use the constant in the function